docs/attributes-registry/gen-ai.md

```diff
@@ -0,0 +1,48 @@
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+# Large Language Model (LLM)
+
+<!-- toc -->
+
+- [Generic LLM Attributes](#generic-llm-attributes)
+  - [Request Attributes](#request-attributes)
+  - [Response Attributes](#response-attributes)
+  - [Event Attributes](#event-attributes)
+
+<!-- tocstop -->
+
+## Generic LLM Attributes
+
+### Request Attributes
+
+<!-- semconv registry.llm(omit_requirement_level,tag=llm-generic-request) -->
+| Attribute  | Type | Description  | Examples  |
+|---|---|---|---|
+| `gen_ai.llm.request.max_tokens` | int | The maximum number of tokens the LLM generates for a request. | `100` |
```

Comment:
Most attributes here should *not* have the `llm` namespace so that we can more easily compose for different span types in the future (image generation, embeddings, etc). For example, `model` will almost always by applicable to all gen_ai spans.  For attributes like `temperature` we can make them optional when applicable for a particular span type (ChatCompletions, etc).

Adding the extra namespace will become unruly and lead to a lot of duplicated attributes IMO.